csrc/anemone_mempool.h

```diff
@@ -10,6 +10,10 @@
 #define ANEMONE_MEMPOOL_WHEN_DEBUG(f)
 #endif
 
+// Officually, the documentation says that un-aligned SSE accesses require
+// 16 byte alignment, but testing suggests 8 byte alignment is sufficient.
```

Comment:
I think this is because we're not using the unaligned instructions directly, `gcc` is generating them.

`gcc` is assuming that our `int64_t` array is 8-byte aligned, so it inserts a preamble which makes sure the memory it operates over is 16-byte aligned.

So, I'm not so sure that the 16-byte alignment case is actually useful. We'll never be able to guarantee 16-byte alignment in all situations, because we might slice out a sub-array which breaks the 16-byte alignment.